omni-relayer/src/workers/near.rs

```diff
@@ -206,3 +332,77 @@ pub async fn claim_fee(redis_client: redis::Client, connector: Arc<OmniConnector
         .await;
     }
 }
+
+pub async fn sign_claim_native_fee(
+    config: config::Config,
+    redis_client: redis::Client,
+    connector: Arc<OmniConnector>,
+) -> Result<()> {
+    let redis_connection = redis_client.get_multiplexed_tokio_connection().await?;
+
+    loop {
+        let mut redis_connection_clone = redis_connection.clone();
+        let Some(events) = utils::redis::get_events(
+            &mut redis_connection_clone,
+            utils::redis::NEAR_FIN_TRANSFER_EVENTS.to_string(),
```

Comment:
Bad naming. 

When I read this code I feel like you ignore ClaimFeeEvent and handle the FinTransferEvent with nonce != 0 as well.